pkgs/tools/backup/mydumper/default.nix

```diff
@@ -32,6 +42,16 @@ stdenv.mkDerivation rec {
     "-DWITH_ZSTD=ON"
   ];
 
+
+  env.NIX_CFLAGS_COMPILE = "-Wno-unused-result";
+
+  postPatch = ''
+    substituteInPlace CMakeLists.txt\
+      --replace-fail "/etc" "$out/etc" \
```

Comment:
the install script installs `/etc/mydumper.cnf`

I only fixed the place where the install script installs, but I could also do it (via a patch) where the source code installs the file?